content/docs/code-splitting.md

```diff
@@ -115,7 +115,7 @@ parse the dynamic import syntax but is not transforming it. For that you will ne
 
 > Note:
 >
-> `React.lazy` and Suspense is not yet available for server-side rendering. If you want to do code-splitting in a server rendered app, we still recommend [React Loadable](https://github.com/thejameskyle/react-loadable). It has a nice [guide for bundle splitting with server-side rendering](https://github.com/thejameskyle/react-loadable#------------server-side-rendering).
+> `React.lazy` and Suspense is not yet available for server-side rendering. If you want to do code-splitting in a server rendered app, we recommend [Loadable Components](https://github.com/smooth-code/loadable-components) or [React Universal Component](https://github.com/faceyspacey/react-universal-component).
```

Comment:
What's the difference between them?

It looks like `react-universal-component` depends on legacy context. While this is true, I'd prefer that we don't recommend it.

Reply:
Handling of CSS imports (including HMR) is much nicer in `react-universal-component`. They're otherwise pretty similar. Since that's a minor difference, I'll drop it in favor of just `loadable-components` and add a link to their SSR section.